workers/advisor/src/main/kotlin/advisor/AdvisorComponent.kt

```diff
@@ -84,7 +84,8 @@ class AdvisorComponent : EndpointComponent<AdvisorRequest>(AdvisorEndpoint) {
             if (response != null) publisher.publish(OrchestratorEndpoint, response)
         }
 
-        EndpointHandlerResult.CONTINUE
+        val handleSingleJob = configManager.config.getBoolean("advisor.handleSingleJob")
```

Comment:
Maybe better call this `singleJobOnly`? That would emphasize that they cannot `also* process single jobs, but *only* those.

Reply:
I think I'd like to keep "handle", what about `handleSingleJobOnly`?

Reply:
Yes, a bit longer, but telling.

Reply:
Done.